docs/content/config/setup.sh.md

```diff
@@ -1,123 +1,26 @@
 ---
-title: Your best friend setup.sh
+title: About setup.sh
 hide:
   - toc
 ---
 
-[`setup.sh`][github-file-setupsh] is an administration script that helps with the most common tasks, including initial configuration. It is intended to be run from the host machine, _not_ from inside your running container.
+!!! note
 
-The latest version of the script is included in the `docker-mailserver` repository. You may retrieve it at any time by running this command in your console:
+    `setup.sh` is not required since v10.2.0. We encourage you to use `docker exec -ti <CONTAINER NAME> setup` instead.
```

Comment:
If you want to keep this, that's fine.

I have made commits to your PRs which removed this admonition from other doc pages however. Changelog for v12 could probably express the information just as well.

---

This is clarified further down, so mostly redundant. If you'd like to keep it, maybe drop the version mention?

```suggestion
    `setup.sh` is not required. We encourage you to use `docker exec -ti <CONTAINER NAME> setup` instead.
```